com.microsoft.mrtk.input/Subsystems/HandsAggregator/MRTKHandsAggregatorSubsystem.cs

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using Microsoft.MixedReality.OpenXR;
```

Comment:
I don't think this package has an explicit dependency on the MR OpenXR package, so I'm not sure if we want to use this here

Reply:
Fixed to use the `TotalJoints` enum